### ISSUE TITLE
Pretty-print the contents of local variables in the Local Variables view

### ISSUE THREAD

**Issue by theres-waldo** [Pernosco/pernosco#11]
While the little arrows help a bit, the contents of a local variable in the Local Variables view is still pretty hard to parse if the variable is a structure with fields that are themselves structures etc.

Pretty-printing the value the way gdb / rr does with `set print pretty on` would be a nice improvement.

**Comment by theres-waldo** [comment]
To illustrate, compare the output of the same variable in the gdb view:

![gdb-view](https://user-images.githubusercontent.com/6721837/70571225-664d1600-1b6b-11ea-8b89-e1ba936f03c7.png)

to the locals view:

![local-view](https://user-images.githubusercontent.com/6721837/70571226-664d1600-1b6b-11ea-8916-1e9a4365b9de.png)

Note also the gdb view puts in enumerator names, that's a nice touch.

**Comment by theres-waldo** [comment]
This issue remains the primary reason I continue reaching for the gdb view. Running `print localVar->mThis->mThat->mOtherThing` is just so much quicker than visually scanning the non-indented text blob in the Local Variables view for those members.

**Comment by khuey** [comment]
We're aware that traversing complex object graphs stills sucks. You can (as of a couple months ago) put `localVar->mThis->mThat->mOtherThing` in the search bar to evaluate that expression without dropping into gdb though.

**Comment by khuey** [comment]
This has been revamped significantly and I'm going to call it finished. Please file new issues if you find additional things that annoy you.
